### ISSUE TITLE
`selector-no-redundant-nesting-selector` issue

### ISSUE THREAD

**Issue by ntwb** [kristerkari/stylelint-scss#81]
[In the docs](https://github.com/kristerkari/stylelint-scss/tree/master/src/rules/selector-no-redundant-nesting-selector#selector-no-redundant-nesting-selector) for the `selector-no-redundant-nesting-selector` rule the very last example **not** considered a warning:

```css
p {
  & + .foo {}
}
```

In the tests [this is expected to be rejected](https://github.com/kristerkari/stylelint-scss/blob/master/src/rules/selector-no-redundant-nesting-selector/__tests__/index.js#L191):

```js
    code: `
      p {
        & + .foo {}
      }
    `,
    line: 3,
    message: messages.rejected,
    description: "when an ampersand precedes a sibling operator",
```

Either the documented example is incorrect, or the test is incorrect.

**Comment by dryoma** [comment]
It's the docs, thanks for reporting!
